staging/src/k8s.io/client-go/examples/README.md

```diff
@@ -0,0 +1,36 @@
+# client-go Usage Examples
+
+This directory contains examples that cover examples of various use cases
+and functionality for Go client library for Kubernetes.
+
+### Authentication
+
+- [**Authenticate in cluster**](./authenticate-in-cluster)
+- [**Authenticate out of cluster**](./authenticate-out-of-cluster)
+
+### Basics
+
+- [**Managing resources with API**](./create-update-delete-deployment): Create,
+  get, update, delete a Deployment resource.
+
+### Advanced Concepts
+
+- [**Third-party resources**](./third-party-resources): Register a custom
+  resource type with the API, create/update/query this custom type, and write a
+  controller to handle events for this custom resource type.
+- [**Work queues**](./workqueue): Create a custom controller with a Watch for Pod events,
```

Comment:
How about "Create a hotloop-free controller with the rate-limited workqueue and the informer framework", i don't think we need to mention "Watch for Pod".

And could you move this example before the tpr one? The latter is more complex.